src/pages/study-page/study-page.tsx

```diff
@@ -16,41 +18,54 @@ export const StudyPage = () => {
   const { getCardsByDeckId } = useCardsClient();
   const [deck, setDeck] = useState<Deck | undefined>();
   const { activeCardKey, activeCardSide, startLesson, pauseLesson, resumeLesson } = usePlayLesson();
+  const [count, setCount] = useState(-2);
```

Comment:
it's just a dummy value here. Since there is a clamp, it shows up as zero, and only when the first card is finished do we see some progress on the bar.